src/client/js/Widgets/DiagramDesigner/ConnectionRouteManager3.js

```diff
@@ -576,5 +583,31 @@ define([
         return newBox;
     };
 
+    ConnectionRouteManager3.prototype.download = function (filename, data) {
+        if (!this.readyToDownload) {
```

Comment:
Will this.readyToDownload ever be true? I can't see where it is ever set to true (at least not in the diff).